build_tfm.py

```diff
@@ -426,6 +432,60 @@ def _check_and_copy(list_of_items, path):
                 _check_and_copy(tf_regression_data["dualcpu"], ROOT)
 
 
+def _copy_psa_libs(source, destination, args):
+    """
+    Copy TF-M binaries from source to destination
+
+    :param source: directory where TF-M binaries are available
+    :param destination: directory to which TF-M binaries are copied to
+    :param args: Command-line arguments
+    """
+
+    output_dir = join(destination, "test", "lib", "TOOLCHAIN_" + TC_DICT[args.toolchain])
```

Comment:
```suggestion
    output_dir = join(destination, "test", "lib", "TOOLCHAIN_" + TC_DICT[args.toolchain])
    
    if not isdir(output_dir):
        os.makedirs(output_dir)
```
Unlike `mkdir`, `makedirs` also creates any parent directories.